ProcessLib/HeatTransportBHE/CreateHeatTransportBHEProcess.cpp

```diff
@@ -147,17 +144,13 @@ std::unique_ptr<Process> createHeatTransportBHEProcess(
         MaterialPropertyLib::createMaterialSpatialDistributionMap(media, mesh);
 
     // find if bhe uses python boundary condition
-    auto const isUsingPythonBC =
+    auto const is_using_python_bc =
         visit([](auto const& bhe) { return bhe.usePythonBC; }, bhes[0]);
```

Comment:
same for the `usePythonBC`.

Reply:
Fixed in 1c511a0027102b9e1599687926abccbd6c63c9b1.